client/app/utils/places.js

```diff
@@ -0,0 +1,151 @@
+/*
+ Generic utilities for dealing with Google Maps place objects.
+*/
+
+const toRadians = (degrees) => degrees * (Math.PI / 180); // eslint-disable-line no-magic-numbers
+
+/**
+ * Calculate the distance between two points.
+ *
+ * @param {google.maps.LatLng} pointA - first point
+ * @param {google.maps.LatLng} pointB - second point
+ *
+ * @return {Number} distance between the given points (in km?)
+ */
+const computeScale = (pointA, pointB) => {
```

Comment:
The naming of this function was my failure, it should be fixed :) It actually doesn't do what the document promises (I think?), but instead returns `distance / 2`. I suggest this should be changed to `getDistance` and then the `/ 2` be done elsewhere.